cmd/pbstatuser/main.go

```diff
@@ -41,7 +43,29 @@ type SourceInfo struct {
 	Identity identity.XRHID
 }
 
+const (
+	metricNameSourceAvailabilityCheckRequestTotal    = "source_availability_check_request_total"
+	metricNameSourceAvailabilityCheckRequestDuration = "source_availability_check_request_duration"
+)
+
 var (
+	totalAvailabilityCheckReqs = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name:        metricNameSourceAvailabilityCheckRequestTotal,
+			Help:        "availability Check requests count partitioned by type (aws/gcp/azure), source status and error",
+			ConstLabels: prometheus.Labels{"service": version.PrometheusLabelName},
+		},
+		[]string{"type", "status", "error"},
+	)
+
+	availabilityCheckReqsDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:        metricNameSourceAvailabilityCheckRequestDuration,
+		Help:        "Availability check Request duration partitioned by type and error",
```

Comment:
Prometheus is not time-aware, please always include the scale in the help - in this case milliseconds.